core/cli/src/rc-file.ts

```diff
@@ -44,6 +46,28 @@ export async function loadToolKitRC(logger: Logger, root: string, isAppRoot: boo
   }
 
   const config: RawRCFile = result.config
+
+  // if a toolkitrc contains a non-empty options field, but not options.{plugins,tasks,hooks},
+  // assume it's an old-style, plugins-only options field.
+  // TODO:KB:20240410 remove this legacy options field handling in a future major version
+  if (
+    config.options &&
+    Object.keys(config.options).length > 0 &&
+    !(config.options.plugins || config.options.tasks || config.options.hooks)
+  ) {
+    config.options = {
+      plugins: config.options as { [id: string]: Record<string, unknown> }
+    }
+
+    logger.warn(
+      `plugin at ${s.filepath(path.dirname(root))} has an ${s.code(
+        'options'
+      )} field that only contains plugin options. these options should be moved to ${s.code(
+        'options.plugins'
+      )}.`
```

Comment:
**suggestion**: clarify that some options will have to be moved to `options.tasks` instead. I think  it will be surprising if a user still gets errors after following the suggestion here. I've tried my best to make it less surprising though it's hard to write something clear for this 😅

```suggestion
      )} field that only contains plugin options. these options should be moved to ${s.code(
        'options.plugins'
      )} (or ${s.code('options.tasks')} if you see subsequent errors.)`
```

Reply:
actually, as well as this warning, with https://github.com/Financial-Times/dotcom-tool-kit/commit/cb98a854a582fe6cb76a5c57156ff5101afcb1bf they'll also have got a bunch of schema errors already. i think it's fine?

Reply:
will they see those schema errors before they move to `options.plugins`?

Reply:
yes